internal/pkg/skuba/addons/cilium.go

```diff
@@ -79,9 +81,9 @@ func renderCiliumPreflightTemplate(addonConfiguration AddonConfiguration) string
 
 type ciliumCallbacks struct{}
 
-func (ciliumCallbacks) beforeApply(addonConfiguration AddonConfiguration, skubaConfiguration *skuba.SkubaConfiguration) error {
+func (ciliumCallbacks) beforeApply(client clientset.Interface, addonConfiguration AddonConfiguration, skubaConfiguration *skuba.SkubaConfiguration) error {
 	ciliumVersion := kubernetes.AddonVersionForClusterVersion(kubernetes.Cilium, addonConfiguration.ClusterVersion).Version
-	client, config, err := kubernetes.GetAdminClientSetWithConfig()
+	_, config, err := kubernetes.GetAdminClientSetWithConfig()
 	if err != nil {
 		return errors.Wrap(err, "unable to get admin client set")
```

Comment:
The error message is slightly misleading now since it's now only getting the config and not the clientset